.travis.yml

```diff
@@ -15,16 +15,14 @@ env:
 
 matrix:
   include:
-    - env: RUNTIME=3.6 TOOLKITS="null pyside2"
-    - env: RUNTIME=3.6 TOOLKITS="pyqt"
+    - env: RUNTIME=3.6 TOOLKITS="null pyqt pyqt5 pyside2"
```

Comment:
don't we need to mark `pyqt5` as an allowed failure anymore?

Reply:
The pyqt5 build was allowed to fail as a result of #815 (https://github.com/enthought/traitsui/issues/821#issuecomment-649736045)

I don't remember the exact reason why they are linked. The tests are passing now with Qt5 so it would be good to move it out of the allow-to-fail group.

Reply:
👍